Tools/TestWebKitAPI/Tests/WebKitGLib/TestWebKitSettings.cpp

```diff
@@ -77,11 +77,6 @@ static void testWebKitSettings(Test*, gconstpointer)
     webkit_settings_set_enable_frame_flattening(settings, TRUE);
     g_assert_true(webkit_settings_get_enable_frame_flattening(settings));
 
-    // Java is enabled by default.
-    g_assert_true(webkit_settings_get_enable_java(settings));
-    webkit_settings_set_enable_java(settings, FALSE);
-    g_assert_false(webkit_settings_get_enable_java(settings));
```

Comment:
Instead of removing the test, I would make it check that the setting is always disabled regardless of calls to `webkit_settings_set_enable_java()` trying to enable it.